core/src/test/java/fr/sncf/osrd/envelope_sim_infra/MRSPTest.java

```diff
@@ -50,7 +50,7 @@ public void setUp() throws IOException, URISyntaxException {
         var speedSection1 = new RJSSpeedSection(NAME, SPEED1, Map.of(TRAIN_TAG1, SPEED2),
                 List.of(new RJSApplicableDirectionsTrackRange("TA0", ApplicableDirection.BOTH,
                         POSITION1, POSITION2)));
-        var speedSection2 = new RJSSpeedSection(NAME, 0, Map.of(TRAIN_TAG2, SPEED3),
+        var speedSection2 = new RJSSpeedSection(NAME, Double.POSITIVE_INFINITY, Map.of(TRAIN_TAG2, SPEED3),
```

Comment:
```suggestion
        var speedSection2 = new RJSSpeedSection(NAME, null, Map.of(TRAIN_TAG2, SPEED3),
```

Reply:
RJSSpeedSection expect a `double` so I can't instanciate with `null`, Do I change the definition to use `Double` instead ?